installer.py

```diff
@@ -392,7 +383,7 @@ def ensure_winetricks_executable(app=None):
         app=app
     )
 
-    if not os.access(config.WINETRICKSBIN, os.X_OK):
+    if config.WINETRICKSBIN == 'Download' or not os.access(config.WINETRICKSBIN, os.X_OK):
```

Comment:
I think we need something else here. If the user provides a preset config and we find that it is not executable, we override it rather than attempting to correct it by making it executable or asking the user what to do. What do you think?

Reply:
I feel uneasy about this change because surely the `WINETRICKSBIN` variable itself should never actually be "Download", right? Rather, if the UI defaults to "Download" while preparing the install, then it seems like *it* should set the correct subsequent `WINETRICKSBIN` value rather than the `install` module having to react to that placeholder value.

Or, coming from another direction, the variable *could* simply be just "Download" or "System", rather than giving a file path. Is it really that useful to allow an arbitrary path for `WINETRICKSBIN`? If it's "Download", then the script would just interpret the location as `INSTALLDIR/data/bin/winetricks`; if it's "System", then the script finds its path with `shutil.which('winetricks')`. That would simplify the code a decent amount, right? I know it would require reworking a few other things, but maybe a better long-term solution for easier maintainability?

Reply:
I was thinking over this one and I went with the smallest hammer (get existing method working).

I agree this needs to be redone. Do you want that redone in this PR or a new one?

Reply:
I've created Issue #170  to address this, rather than forcing it into this PR.